src/Controls/tests/Core.UnitTests/WindowsTests.cs

```diff
@@ -72,7 +72,7 @@ public class TestApp : Application
 			public TestWindow CreateWindow() =>
 				(TestWindow)(this as IApplication).CreateWindow(null);
 
-			protected override IWindow CreateWindow(IActivationState activationState)
+			protected override Window CreateWindow(IActivationState activationState)
```

Comment:
The `Windows` property is now a list of Control's Window as that is probably what we want at this level.